practice-app/practice/cron.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import requests
+from api.models import Equipment, Parity
+from django_cron import CronJobBase, Schedule
+from django.utils.timezone import make_aware
+from datetime import datetime
+
+
+class GetExchangeRates(CronJobBase):
+    RUN_EVERY_MINS = 5
+
+    schedule = Schedule(run_every_mins=RUN_EVERY_MINS)
+    code = 'practice.GetExchangeRates'
+
+    def do(self):
+
+        equipments = Equipment.objects.all()
+
+        symbols = [equipment.symbol for equipment in equipments]
+
+        if "EUR" in symbols:
+            symbols.remove("EUR")
+            symbols_as_joined_without_eur = ",".join(symbols)
+            symbols.append("EUR")
+
+        symbols_as_joined = ",".join(symbols)
+
+        for base_symbol in symbols:
+
+            print("-----------", base_symbol, "-----------")
+
+            try:
+                response = requests.get("https://api.exchangeratesapi.io/latest?base={}&symbols={}".format(
+                    base_symbol,
+                    symbols_as_joined if base_symbol != "EUR" else symbols_as_joined_without_eur)
+                )
+            except requests.exceptions.RequestException as e:
+                print(e)
+                sys.exit(1)
```

Comment:
maybe instead of sys.exit, we can return the corresponding status code?

Reply:
Users do not interact with this class, it works in the background. If something goes wrong, it will print the exception to the console. We may print status code as well, it will be helpful. However, sys.exit(1) should exist, because an error occurs we shouldn't continue to execute. 😊😊